Dockerfile

```diff
@@ -1,16 +1,19 @@
-FROM debian:jessie
+FROM debian:buster-slim
+RUN apt-get update && apt-get dist-upgrade -y && \
+    apt-get install -y ca-certificates git curl jq build-essential libluajit-5.1-2 luarocks python3-setuptools python3-dev python3-pip
+RUN luarocks install lua-cjson
 
-RUN apt-get update -y -qq && apt-get install -y ca-certificates curl wget build-essential libluajit-5.1-2 libluajit-5.1-dev luarocks python3-setuptools python3-dev libpython3.4 locales ruby-dev gcc make rubygems rpm gnupg expect perl git openssh-client procps autoconf autogen libtool
+RUN pip3 install grpcio
+RUN pip3 install protobuf
 
-RUN wget -O go.tgz "https://dl.google.com/go/go1.10.3.linux-amd64.tar.gz" && tar -C /usr/local -xzf go.tgz && rm go.tgz
-RUN mkdir -p "/go/src" "/go/bin" && chmod -R 777 "/go"
+# Go install
+RUN curl -sL https://dl.google.com/go/go1.12.8.linux-amd64.tar.gz | tar -xzC /usr/local/
 
-RUN luarocks install lua-cjson
+RUN mkdir -p /go/src/plugin-build
+COPY data/build.sh /build.sh
+RUN chmod +x /build.sh
 
-# Install fresh pip to support binary distributions
-RUN wget https://bootstrap.pypa.io/get-pip.py && python3 get-pip.py
-RUN pip3 install grpcio
+ENV PATH=$PATH:/usr/local/go/bin
+ENV GOPATH=/go
 
-RUN GOPATH=/ /usr/local/go/bin/go get github.com/mitchellh/gox
-RUN gem install fpm && gem install rake && gem install package_cloud
-RUN mkdir -p ~/rpmbuild/SOURCES ~/rpmbuild/SPECS
+ENTRYPOINT ["/build.sh"]
```

Comment:
Do I understand right that `build.sh` doesn't have any params here? there is no `CMD` here so I came up with this question: how do we build plugin for some specific gateway release tag?

Reply:
My understanding is that the docker image tag will track that of the gateway.

Reply:
@dencoded I had omitted to handle vendored modules. This has been addressed in the latest push.